### PR TITLE
Do not list dependencies being checked for CVEs.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-3045" title="DX-3045" target="_blank">BugDX-3045</a>  CVE report doesn't address updated package
  </td></table>
  <br />
 

Similar to how we're no longer listing which packages are being installed.

I changed some of the messaging because originally it assumed only one package at a time could be installed (and added, not updated).

Samples:

```
% state install urllib3@2.0.2
█ Installing Package

Operating on project qam/newpub, located at /path/to/newpub.

• Searching for packages in the ActiveState Catalog ✔ Found
• Resolving Dependencies ✔ Done
• Checking for vulnerabilities (CVEs) in your project and its dependencies x Unsafe

  Warning: Found 2 new vulnerabilities (CVEs). Severity: 1 High, 1 Medium. Run 'state security' for more info.

Added: language/python/urllib3@2.0.2

Your local project has been updated.
Run state push to save changes to the platform.
```

```
% state install language/python@3.11.9 pytest
█ Installing Package

Operating on project qam/newpub, located at /path/to/newpub.

• Searching for packages in the ActiveState Catalog ✔ Found
• Resolving Dependencies ✔ Done
• Checking for vulnerabilities (CVEs) in your project and its dependencies x Unsafe

  Warning: Found 2 new vulnerabilities (CVEs). Severity: 2 High. Run 'state security' for more info.

Updated: language/python@3.11.9
Updated: language/python/pytest@Auto

Your local project has been updated.
Run state push to save changes to the platform.
```

```
% state install private/ActiveState-CLI-Testing/language/python/django_dep --ts=2024-09-10T16:36:34.393Z
█ Installing Package

Operating on project qam/newpub, located at /path/to/newpub.

• Searching for packages in the ActiveState Catalog ✔ Found
• Resolving Dependencies ✔ Done

  Installing django_dep@1.0.0 includes 1 direct dependencies, and 3 indirect dependencies.
  └─ django@4.2 (3 dependencies)

• Checking for vulnerabilities (CVEs) in your project and its dependencies x Unsafe

  Warning: Found 9 new, indirect vulnerabilities (CVEs)

  • 2 Critical: django@4.2: CVE-2023-31047, CVE-2024-42005
  • 7 High: django@4.2: CVE-2023-36053, CVE-2023-41164, CVE-2023-43665, CVE-2024-24680, CVE-2024-41989, 
    CVE-2024-41990, CVE-2024-41991

  For more information on these vulnerabilities run 'state security open <ID>'.
  To disable prompting for vulnerabilities run 'state config set security.prompt.enabled false'.

Do you want to continue installing this dependency despite its vulnerabilities? (y/N)
```

### CODE REVIEW

**Comment by mitchell-as** [comment]
The CVE prompt test failure is being tracked in DX-3064.

The other failures are known and unrelated to this PR.

File: internal/config/instance.go [hunk 1]
```diff
@@ -185,7 +185,11 @@ func (i *Instance) Get(key string) interface{} {
 		return result
 	}
 	if opt := mediator.GetOption(key); mediator.KnownOption(opt) {
-		return opt.Default
+		default_ := opt.Default
+		if enum, ok := default_.(*mediator.Enums); ok {
+			return enum.Default
+		}
+		return default_
```
Comment (Naatan) [inline internal/config/instance.go:192]: This ought to be addressed at the type level, or in the mediator. Handling it here sprinkles the business logic for handling these types across three packages.

**Review by Naatan — CHANGES_REQUESTED** [review]
I don't feel like it's necessary to make these changes to CVEs or summaries. It was done for install because in terms of solving we only solve what needs solving, which isn't necessarily all user input. Here that is not a factor.

Also, the jira story is a bug, I'd prefer that if we find we need to make functional / UI/UX changes like this we first bounce it back to triage.

**Comment by mitchell-as** [comment]
Closing. Will either try again in a new PR, or will put this back in triage.

**Pull request closed without merging**